driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -241,32 +241,44 @@ public Config toConfig()
     }
 
     /**
-     * A configuration to configure TLS authentication
+     * Control the level of encryption to require
+     */
+    public enum EncryptionLevel
+    {
+        /** With this level, the driver will only connect to the server if it can do it without encryption. */
+        REJECTED,
```

Comment:
Agreed. Not even sure we need two levels of settings (security on/off + security mode). We should probably be able to make do with something like SECURITY_NONE, SECURITY_TOFU and SECURITY_FULL.